commcare_connect/opportunity/views.py

```diff
@@ -1236,7 +1236,7 @@ def invoice_approve(request, org_slug, pk):
 @csrf_exempt
 def user_invite_delete(request, org_slug, opp_id, pk):
     opportunity = get_opportunity_or_404(opp_id, org_slug)
-    invite = get_object_or_404(UserInvite, pk=pk, opportunity=opportunity)
+    invite = get_object_or_404(UserInvite, pk=pk, opportunity=opportunity, status=UserInviteStatus.not_found)
```

Comment:
I don't think we want to 404 here if the invite has a different status. 404 means we couldn't find the invite, but thats not the case here. At worst we want to 403, since they aren't allowed, but if this can actually be triggered by a user, we should probably return a normal response with a message telling them why the delete failed.

Reply:
I have added a 403 response on this view when the user invite is not ok for deletion. The user cannot trigger this for cases where the invites are valid as we hide the button.